inc/functions.php

```diff
@@ -5156,7 +5156,7 @@ function convert_through_utf8($str, $to=true)
 }
 
 /**
- * DEPRECATED! Please use other alternatives.
+ * @deprecated deprecated since version 1.8.4 Please use other alternatives.
```

Comment:
Should be `1.8.6` now. Also note that this is an invalid doc block as per #2026 and will be changed in that PR.